lib/tasks/report_inconsistent_facet_values.rake

```diff
@@ -0,0 +1,57 @@
+require "gds_api/rummager"
+require "json"
+
+class EmptyFacetValueError < StandardError
+end
+
+desc "Check for and report any non-expanded facet values"
+task :report_inconsistent_facet_values do
+  # List of facets to check
+  facets = %w{
```

Comment:
I've added `document_collections` and `specialist_sectors` but the other ones (`document_series`, `organisation_content_ids` and `topic_content_ids`) are not valid facet fields.

For example, an error from rummager:

`{"error":"\"document_series\" is not a valid facet field"}`